app/controllers/api_resource_comments_controller.rb

```diff
@@ -4,4 +4,46 @@ def show
     comments = Resource.find(resource_id)&.resource_comments
     render json: comments, only: %i[id content resource_id user_id]
   end
+
+  def create
+    content = params['content']
+    resource_id = params['resource_id']
+    comment = ResourceComment.new(content:, user: current_user, resource_id:)
+
+    if comment.save
+      render_successful_response(comment)
+    else
+      handle_save_error(content, resource_id)
+    end
+  end
+
+  private
+
+  def render_successful_response(new_comment)
+    status = :created
+    render json: {
+      'commentId': new_comment.id,
+      'content': new_comment.content,
+      'resource_id': new_comment.resource.id
+    }, status:
+  end
+
+  def handle_save_error(content, resource_id)
```

Comment:
I have trouble with these methods. The first thing is if they are exclusive to the ApiResourceCommentsController or they belong to a superior class. I think that is the second case.  It could be better to have an Error class.